resize-observer/devicepixel2.html

```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<link rel="match" href="devicepixel2-ref.html">
+<meta name="assert" content="Device pixel content box sizes and pixel snapping are correct in Resize Observer callback">
+<style>
+  html, body {
+    margin: 0;
+    height: 100%;
+  }
+  #back {
+    width: 100%;
+    height: 100%;
+  }
+  #front {
+    position: absolute;
+    left: 0;
+    top: 0;
+  }
+  .outer {
+    width: 99.5px;
```

Comment:
99.5px feels unnecessarily-magic; consider changing this to 100px (or even 127px [same ballpark, but prime].)

I think your intent is that 99.5px is "weird" and likely to produce bugginess, but 100px actually produces more fully-black areas (and it does so sooner, i.e. in higher-up rows, than 99.5px does on my system at least -- Firefox Nightly with devicePixelRatio = 1)

The fractional size here implicitly suggests that we need a fractional size to trigger the bug in question; but in fact we don't.

Reply:
fixed